src/screens/ShowWorkoutScreen/ShowWorkoutScreen.js

```diff
@@ -1,6 +1,31 @@
 import React, { useState, useEffect } from 'react';
 import axios from 'axios';
-import { ScrollView, StyleSheet, Text, View } from 'react-native';
+import { ScrollView, StyleSheet, Text, TouchableHighlightBase, View, TouchableOpacity} from 'react-native';
+import { EditWorkoutButton } from '../../SVGs'
+
+  const ExerciseList = (props) => {
+    const { exercises = [], getWorkout } = props
+    console.log(exercises)
+    var titles = [];
+    exercises.forEach((exercise) => {
+      titles.push({'title': exercise.title, 'duration': exercise.duration, 'description': exercise.description })
+    });
```

Comment:
Unsure why titles needs to be created, it has the same structure as exercises without the image